### DISCUSSION TITLE
Ajout des deux couleurs vacances et jours fériés à la légende

### DISCUSSION THREAD

**Discussion by gbwin** [JeromeDevome/GRR#193] [Idées]
Bonjour,
je viens d'activer la fonctionnalité permettant l'affichage des vacances et jours fériés.
Je vois maintenant apparaitre un bandeau (vert ou rouge) sur toute la largeur de la page d'accueil.
Surtout pour les bandeaux rouges, mes usagers vont me posser la question "qu'est ce que cela veut dire ?"
Serait-il possible d'ajouter cette information dans la légende (ou sur la barre colorée elle même, si c'est plus simple)
Merci d'avance.
Bonne journée.
GBwin

**Comment by ynaessens** [comment]
Bonjour,
mettre ces couleurs dans la légende ne risque-t-il pas d'entrer en conflit avec des couleurs attribuées  un type de réservation ?
Je propose plutôt d'activer un "title" sur la date du jour férié ou de vacances rappelant cette attribut. Qu'en pensez-vous ?
Cordialement,
YN

**Reply by gbwin** [reply]
Bonjour,
justement c'est ce que je me disais aussi que ça n'allait pas trop avec.
Ah oui, c'est une bonne idée le title sur la date, effectivement 👏😉
GBwin
